src/operator/toArray.ts

```diff
@@ -3,6 +3,24 @@ import { Subscriber } from '../Subscriber';
 import { Observable } from '../Observable';
 
 /**
+ * Creates an array containing all emissions of an Observable.
```

Comment:
Seems sane to simplify, i'll update the wording a bit. 

One thing that i found odd; when compiling the documentation I get a marble diagram image in `/tmp/docs/img/toArray.png` but it only shows the marbles without text. Is that somehow post-processed when releasing the documentation?